README.rst

```diff
@@ -257,7 +257,27 @@ Since going over all (or select) keys in an S3 bucket is a very common operation
   ...     print(key, round(len(content) / 2**20))
   annual/monthly_rain/2010.monthly_rain.nc 13
   annual/monthly_rain/2011.monthly_rain.nc 13
-  annual/monthly_rain/2012.monthly_rain.nc 13
+  annual/monthly_rain/2012.monthly_rain.nc 13 
+
+Specific s3 object version
+--------------------------------------
+
+The ''open'' function has the parameter version_id, which allows you to get the desired version of the file from bucket S3.
```

Comment:
This is a bit misleading. You're talking about the smart_open.s3.open function, not the smart_open.open function (but you're importing "from smart_open import open", which is even more misleading). While what you say is correct, nobody really uses smart_open.s3.open, because it's mostly an internal function. everybody uses smart_open.open (or the older smart_open.smart_open).

Please rewrite these instructions and use the top-level function in your examples.